Improves configuration docs for Admin (#61)
kosgrz/mattermost-plugin-bitbucket: docs/admin-guide/configuration.md

## Step 1: Register an OAuth Application in Bitbucket

1. Go to [https://bitbucket.org](https://bitbucket.org) and log in.

> 2. From your profile avatar in the bottom left, click on the workspace in the **Recent workspaces** list or click **All workspaces** to open an entire list from which to choose.

Reviewer: ```suggestion
2. From your profile avatar in the bottom left, select the workspace in the **Recent workspaces** list or select **All workspaces** for a full list.
```

>    * Click **Settings** on the left sidebar to open the Workspace settings.

Reviewer: ```suggestion
   * Select **Settings** in the left sidebar to open the workspace settings.
```

>    * Click **OAuth consumers** under **Apps and features** on the left navigation.

Reviewer: ```suggestion
   * Under **Apps and features**, select **OAuth consumers**.
```
